Directory.Build.props

```diff
@@ -3,6 +3,12 @@
     <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
   </PropertyGroup>
 
+  <!-- Properties that need to be available early for SDK targets -->
+  <PropertyGroup Condition="'$(MSBuildProjectName)' == 'Moq.Analyzers'">
+    <Description>Roslyn analyzer that helps to write unit tests using Moq mocking library by highlighting typical errors and suggesting quick fixes. Port of Resharper extension to Roslyn. Find the full list of detected issues at project GitHub page.</Description>
```

Comment:
This seems like a weird property to hoist up. Consider refactoring. This doesn't make sense to be in `Directory.Build.props`. Rather, it makes sense to be in the `Moq.Analyzers.csproj`.